.circleci/config.yml

```diff
@@ -0,0 +1,100 @@
+version: 2
+jobs:
+  build:
+    docker:
+      # base image
+      - image: ubuntu:14.04
+      # statsd
+      - image: kamon/grafana_graphite
+      - image: atlassianlabs/localstack:0.4.0
+        environment:
+          - SERVICES=s3:5000
+          - DEFAULT_REGION=us-east-1
+          - HOSTNAME=localstack-s3
+      - image: postgres:9.4
+        environment:
+          # Create the superuser account
+          - POSTGRES_USER=postgres
+          - POSTGRES_PASSWORD=aPassword
+          - POSTGRES_DB=breakpad
+      - image: elasticsearch:1.4.5
+      - image: rabbitmq:3.6.9
+        environment:
+          - RABBITMQ_DEFAULT_USER=rabbituser
+          - RABBITMQ_DEFAULT_PASS=rabbitpwd
+          - RABBITMQ_DEFAULT_VHOST=rabbitvhost
+    working_directory: /
+    steps:
+      - run:
+          name: Install essential packages
+          command: |
+            apt-get update
+            apt-get install -y ca-certificates curl build-essential make
+
+      - checkout:
+          path: /socorro
+
+      - run:
+          name: Create version.json
+          working_directory: /socorro
+          command: |
+            # create a version.json per https://github.com/mozilla-services/Dockerflow/blob/master/docs/version_object.md
+            printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n' \
+            "$CIRCLE_SHA1" \
+            "$CIRCLE_TAG" \
+            "$CIRCLE_PROJECT_USERNAME" \
+            "$CIRCLE_PROJECT_REPONAME" \
+            "$CIRCLE_BUILD_URL" > /socorro/version.json
+
+      - store_artifacts:
+          path: /socorro/version.json
+
+      # FIXME: should use an image w/ docker installed by default
+      - run:
+          name: Install Docker
+          command: |
+            set -x
+            VER="17.03.0-ce"
+            curl -L -o /tmp/docker-$VER.tgz https://get.docker.com/builds/Linux/x86_64/docker-$VER.tgz
+            tar -xz -C /tmp -f /tmp/docker-$VER.tgz
+            mv /tmp/docker/* /usr/bin
+
+      - run:
+          name: Install Docker Compose
+          command: |
+            set -x
+            curl -L https://github.com/docker/compose/releases/download/1.11.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+            chmod +x /usr/local/bin/docker-compose
+
+      - setup_remote_docker
+
+      - run:
+          name: Build Docker images
+          working_directory: /socorro
+          command: |
+            docker info
+            make dockerbuild
+
+      # - run:
+      #     name: Setup test environment
+      #     working_directory: /socorro
+      #     command: make dockersetup
```

Comment:
dockersetup doesn't set up the test environment--it sets up the environment for running the processor and webapp. We don't need it just to run the tests.